docs/CONFIG-VARS.md

```diff
@@ -431,7 +431,7 @@ The following variables are used to describe the machine targets for the SAS Viy
 
 | Name | Description | Type | Notes |
 |:---|---:|---:|---:|
-| postgres_ip | Static IP address for PostgreSQL server | string | This field is required if you are configuring a PostgreSQL server |
+| postgres_server_name | Name of the PostgresSQL server | string | |
```

Comment:
Hey, I think you should change 'PostgresSQL' to 'PostgreSQL'.

Reply:
Good catch, just updated the spelling to 'PostgreSQL'